database/snowflake/store_query_results.py

```diff
@@ -0,0 +1,95 @@
+import snowflake.connector
+import argparse
+import os
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--username', dest='username', required=True)
+    parser.add_argument('--password', dest='password', required=False)
+    parser.add_argument('--account', dest='account', required=True)
+    parser.add_argument('--database', dest='database', required=True)
+    parser.add_argument('--schema', dest='schema', required=False)
+    parser.add_argument('--query', dest='query', required=True)
+    parser.add_argument('--destination-file-name', dest='destination_file_name',
+            default='output.csv', required=True)
+    parser.add_argument('--destination-folder-name',
+            dest='destination_folder_name', default='', required=False)
+    parser.add_argument('--file-header', dest='file_header', default='True',
+            required=False)
+    args = parser.parse_args()
+    return args
+
+
+def convert_to_boolean(string):
+    """
+    Shipyard can't support passing Booleans to code, so we have to convert
+    string values to their boolean values.
+    """
+    if string in ['True', 'true', 'TRUE']:
```

Comment:
I'd suggest making this a case insensitive equality check to make do for all typos, and not just these three cases. @blakeburch thoughts?

Reply:
I'm fine leaving as is. Majority use will be directly through Shipyard, where the data will always be passed correctly via a Boolean Input or Select Input.